Update README for released version (#22)
conda-incubator/conda-libmamba-solver: README.md

> 1. Install the canary builds of both `conda` and `conda-libmamba-solver` in your `base` environment.

Reviewer: ```suggestion
1. Install `conda-libmamba-solver` in your `base` environment, which should have `conda` 4.12 or greater.
```

> The following channels are required for now:

Reviewer: ```suggestion
```

First make sure you are running conda 4.12.0 or higher:
```bash
conda update -n base conda
```

Then install conda-libmamba-solver:
```bash

> conda install --name=base conda-libmamba-solver

Reviewer: ```suggestion
conda install -n base conda-libmamba-solver
```

For consistency
